src/hooks/useUpdateSkipVerification.tsx

```
@@ -0,0 +1,54 @@
+        queryClient.setQueryData(['user', address], context.previousUser);
+      }
+      console.error('Failed to update skipVerification', err);
+    },
+    onSuccess: (data, variables, context) => {
+      console.log(
+        'skipVerification updated successfully',
+        data,
+        variables,
+        context,
+      );
+      onSuccess && onSuccess();
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)